risc0/zkvm/src/host/api/convert.rs

```diff
@@ -1206,7 +1209,23 @@ impl TryFrom<pb::api::ProveKeccakRequest> for ProveKeccakRequest {
                 .control_root
                 .ok_or_else(|| malformed_err("ProveKeccakRequest.control_root"))?
                 .try_into()?,
-            input: value.input,
+            input,
         })
     }
 }
+
+#[stability::unstable]
+pub(crate) fn keccak_input_to_bytes(input: &[KeccakState]) -> Vec<u8> {
```

Comment:
If something is `pub(crate)` then `unstable` doesn't make sense.

Reply:
well, just wanted to make sure the method wasn't dead code and produce a warning. Perhaps I'm misunderstanding the unstable semantics here.